.github/prompts/summarize-repo-deep-dive.prompt.md

```diff
@@ -0,0 +1,67 @@
+Summarize this Terraform AzureRM provider repository for me. Please provide a comprehensive technical overview that covers:
```

Comment:
I input "generate a new resource file for postgresql flexible server backup". It only generates three files "postgresql_flexible_server_backup_resource.go", "postgresql_flexible_server_backup_resource_test.go" and "parse/postgresql_flexible_server_backup.go". I think it should update "client.go" and "registration.go" and add validation file for the name of the "flexible_server_backup" and resource doc file. This is the PR https://github.com/hashicorp/terraform-provider-azurerm/pull/29201/files I submitted for new resource Postgresql fs backup. I assume the generated result should be align with this PR. Could you take a look?

Reply:
It's not a magic bullet, you need to instruct the AI what you need it to do. After it creates the new resource you will need to ask it to create the client, register the resource, add test and documentation, etc.

Reply:
I thought that after using the prompt file, it had already provided the possible correction information, so the user wouldn’t need to enter more text for fine-tuning. Thanks for the explanation.